animations/spells/1st/force-barrage.json

```diff
@@ -0,0 +1,17 @@
+{
+	"item:slug:force-barrage": [
+		{
+			"trigger": "damage-roll",
+			"preset": "ranged",
+			"file": "jb2a.magic_missile.purple",
+			"options": {
+				"waitUntilFinished": -4000
+			},
+			"contents": [
+				{},
```

Comment:
PF2e expects you to roll damage multiple times, so the animation should only shoot one magic missile at a time.
This also assumes you always cast the 3-action version, which we shouldn't do.

Reply:
Yea i didn't keep that in mind when making it, but that makes sense in hindsight